src/test/java/io/reactivex/internal/operators/observable/ObservableCacheTest.java

```diff
@@ -35,7 +35,7 @@
 public class ObservableCacheTest {
     @Test
     public void testColdReplayNoBackpressure() {
-        ObservableCache<Integer> source = (ObservableCache<Integer>)ObservableCache.from(Observable.range(0, 1000));
+        ObservableCache<Integer> source = (ObservableCache<Integer>)new ObservableCache<Integer>(Observable.range(0, 1000), 16);
```

Comment:
Is the cast really needed now?

Reply:
So the test can access the the package-private methods verifying the internal state.

Reply:
Either I'm missing something or 

```
ObservableCache<Integer> source = (ObservableCache<Integer>)new ObservableCache<Integer>(Observable.range(0, 1000), 16);
```

can just be

```
ObservableCache<Integer> source = new ObservableCache<Integer>(Observable.range(0, 1000), 16);
```